metricbeat/module/docker/_meta/config.yml

```diff
@@ -14,6 +14,9 @@
   # If set to true, replace dots in labels with `_`.
   #labels.dedot: false
 
+  # If set to true, collects metrics per core.
```

Comment:
Will this make the reference file and this one identicaly? Can we remove the reference file afterwards?